addons/binding/org.openhab.binding.modbus/DEVELOPERS.md

```diff
@@ -0,0 +1,111 @@
+# For Developers
+
+## Testing Serial Implementation
+
+You can use test serial slaves without any hardware on linux using these steps:
+
+1. Set-up virtual null modem emulator using [tty0tty](https://github.com/freemed/tty0tty)
+2. Download [diagslave](http://www.modbusdriver.com/diagslave.html) and start modbus serial slave up using this command:
+
+```
+./diagslave -m rtu -a 1 -b 38400 -d 8 -s 1 -p none -4 10 /dev/pts/7
+```
+
+3. Configure openHAB's modbus slave to connect to `/dev/pts/8`:
+
+```
+xxx.connection=/dev/pts/8:38400:8:none:1:rtu
```

Comment:
This seems to be obsolete (refers to openHAB1 binding configuration syntax). Could you remove this code block?

Reply:
ok, removed this block

Reply:
Please remove _just_ the code block but keep the advice on the serial port, e.g.

> Configure openHAB's modbus `serial` thing to connect to `/dev/pts/8`

Reply:
ok, now I got it. Updated.